package.json

```diff
@@ -69,7 +69,7 @@
     "registry": "https://registry.npmjs.org"
   },
   "lint-staged": {
-    "*.{js,ts}": "yarn lint"
+    "*.{js,ts}": "eslint --ext js,ts"
```

Comment:
I'm curious what you think RE: the duplication here. On one hand, I dislike having two spots to specify `eslint` options but on the other, I don't see how to easily share config.

Reply:
I agree 💯 .

Duplication is annoying, but it's pretty minor and easily worth paying for lint speed.